### DISCUSSION TITLE
Share your Formats 🎉

### DISCUSSION THREAD

**Discussion by maharshivpatel** [frappe/print_designer#104] [Show and tell]
### It would be helpful in improving User Experience.

For Incentive, anyone who shares fully designed print formats will be getting priority bug fixes from me 🙈

**Comment by development-rclit** [comment]
Hi Mr Patel, I am always getting error with the tool. How do I get help on this? thanks a lot for the great work

**Reply by maharshivpatel** [reply]
create github issue with information about error.

**Reply by development-rclit** [reply]
thank you! btw do you provide paid support to prioritise issue resolution? many thanks

**Reply by maharshivpatel** [reply]
@development-rclit you can create the issue on github if it is bug i will try to fix it as soon as possible. priority is always defined by how many and how much people are getting affected. 

https://github.com/frappe/print_designer/issues/new

**Reply by development-rclit** [reply]
thanks a lot

**Comment by batonac** [comment]
I just published two print formats that took quite a bit of effort (and a [pull request](https://github.com/frappe/print_designer/pull/466), ha!), but I'm very happy with how they turned out and what's all possible with this tool (in spite of the current paper cuts).

Both are for custom doctypes, the first is a multipage "Sales Proposal" which can be associated with multiple "Sales Quotes":

[<img width="500" alt="image" src="https://github.com/user-attachments/assets/03ffce85-028b-4b52-b0e8-a284a545b398" />](https://github.com/user-attachments/files/22284517/Proposal-0.pdf)

The actual "Sales Quote" is much more concise but does include a page of terms and conditions:

[<img width="500" alt="image" src="https://github.com/user-attachments/assets/3da0d162-a4ed-429a-a35f-75e22702f5ca" />](https://github.com/user-attachments/files/22284471/SQ-20.pdf)
